rfd/0154-logging-guidelines.md

```diff
@@ -211,6 +211,8 @@ to `log/slog`, we should do our best to start following a few simple rules.
    a `context.Background` to indicate as such.
 1) The message should be a fragment, and not a full sentence. Terminating the message with punctuation should be
    avoided.
+1) Starting the message with capitalization is preferred ("Speed threshold reached" instead of "speed threshold
+   reached").
```

Comment:
Ok, this one I definitely disagree - if it starts with a capital letter it ought to be a full phrase and it should have proper punctuation, IMO.

Reply:
I don't mind the particular way it goes, but I would like to have _a_ standard. Capitalized seems to be what the RFD does (most of the time at least).

Reply:
(I prefer the style with lowercase at the beginning of the message and no punctuation, tho)

Reply:
Some quick counts show we seems to favor capitalization too:

(Warning, gross and imprecise script coming up.)

```shell
# total
$ rg 'Info(|f|Context)\(.*?".+?"' --type go | awk -F'"' '{print $2}' | wc -l
> 1012

# Capitalized
$ rg 'Info(|f|Context)\(.*?".+?"' --type go | awk -F'"' '{print $2}' | grep -E '^[A-Z]' | wc -l
> 864

# rg is alias that is somewhat like this:
# /opt/homebrew/bin/rg -. -g=!*.pb.go -g=!*.svg -g=!.git/ -g=!vendor/* 
```

864 if the 1012 info samples are capitalized.